src/components/App/App.tsx

```diff
@@ -207,6 +212,10 @@ export default class App extends React.Component<AppProps, AppState> {
     this.setState({ settings });
   };
 
+  setOwner = (owner: string) => {
```

Comment:
I assume the request that loads this data is still in the Settings tab. . . may be good to relocate it here so it's in the same component?